common/source/docs/common-serial-escs.rst

```diff
@@ -0,0 +1,11 @@
+.. _common-serial-escs:
+
+===========
+Serial ESCs
+===========
+
+ArduPilot supports ESCs connected via a UART port that uses serial protocol. Some also provide motor information telemetry back to the autopilot.
+
+.. toctree::
+
```

Comment:
need to add :maxdepth: 1 to prevent subheadings from appearing immediately below the link to the page...grep examples in the wiki